### PR TITLE
Saga: Add the minimum version info to the component doc generator

### PR DESCRIPTION
**Pull request by eledobleefe**
We add the 'Available from vXX.X.X' in the component doc generator so, whenever a user generates the documentation, it will automatically added to the file.

### CODE REVIEW

**Comment by eledobleefe** [comment]
@Clarity-89, I've also suggested this change in the Google Doc called "[Template] Saga DS Component Documentation". 
By the way, I could not find the documentation on Saga about generating either a template or a component documentation. Is it missed? 🤔

File: scripts/templates/component.mdx.hbs [hunk 1]
```diff
@@ -4,6 +4,8 @@ description: Grafana Labs {{componentName}} component
 ---
 
 # {{componentName}} <Badge text='ready' color='green'></Badge> <StorybookLink path='/story/overlays-alert-toast--basic'/>
+_Available from vXX.X.X_ 
+(Specify the minimum version of Grafana in which the component is available. Needed for plugin devs.)
```
Comment (Clarity-89) [inline scripts/templates/component.mdx.hbs:8]: This should be a comment I assume?

**Comment by Clarity-89** [comment]
> @Clarity-89, I've also suggested this change in the Google Doc called "[Template] Saga DS Component Documentation". By the way, I could not find the documentation on Saga about generating either a template or a component documentation. Is it missed? 🤔

Hmm, yeah, it might be undocumented. I'll need to add it.

File: scripts/templates/component.mdx.hbs (hunk 1 above)
Comment (eledobleefe) [reply scripts/templates/component.mdx.hbs:8]: Yes 😅

File: scripts/templates/component.mdx.hbs [hunk 2]
```diff
@@ -4,6 +4,8 @@ description: Grafana Labs {{componentName}} component
 ---
 
 # {{componentName}} <Badge text='ready' color='green'></Badge> <StorybookLink path='/story/overlays-alert-toast--basic'/>
+_Available from vXX.X.X_ 
+{/* Specify the minimum version of Grafana in which the component is available. Needed for plugin devs.*/}
```
Comment (Clarity-89) [inline scripts/templates/component.mdx.hbs:8]:
```suggestion
[//]: # (Specify the minimum version of Grafana in which the component is available. Needed for plugin devs.)
```
Let's be consistent with the comment format.

**Review by Clarity-89 — APPROVED** [review]

**Pull request merged**